single_level.py

```diff
@@ -16,35 +16,36 @@
 
 spreadsheet = os.path.join(config.get("Destinations", "directory"), config.get("Destinations", "filename"))
 
-#DACS Required Notes to check for
-required_notes = ["scopecontent", "accessrestrict"]
+def get_note_contents(resource, array, note_type):
+	notes = resource["notes"]
+	content_list = []
+	for note in notes:
+		try:
+			if note["type"] == note_type:
+				if note["jsonmodel_type"] == "note_singlepart":
+					content_list.append(note["content"].encode('utf-8'))
+				else:
+					content_list.append(note["subnotes"][0]["content"].encode('utf-8'))
+		except:
+			pass
+	return ", ".join(content_list)
 
-
-def get_note_types(resource):
-	note_types = []
-	for note in resource["notes"]:
-		if note["jsonmodel_type"] == ("note_multipart" or "note_singlepart"):
-			note_types.append(note["type"])
-		else:
-			note_types.append(note["jsonmodel_type"])
-	return note_types
-	
 def get_values(resource, array, value):
 	value_types = []
 	for item in resource[array]:
 		if item[value]:
 			value_types.append(item[value])
 		else:
 			value_types.append("false")
-	return " , ".join(value_types)
-	
+	return ", ".join(value_types)
```

Comment:
Again, we should probably use a more unique character; whatever you use for the above.